internal/csi/csi.go

```diff
@@ -71,3 +82,73 @@ func BinsForRange(start, end uint32, minShift, depth int32) []uint16 {
 func maximumBinWidth(minShift, depth int32) uint32 {
 	return uint32(1 << uint32(minShift+depth*3))
 }
+
+// Read reads index data from csi and returns a set of BGZF chunks covering
+// the header and all mapped reads that fall inside the specified region.  The
+// first chunk is always the BCF header.
+func Read(csiFile io.Reader, region genomics.Region) ([]*bgzf.Chunk, error) {
+	gzr, err := gzip.NewReader(csiFile)
+	if err != nil {
+		return nil, fmt.Errorf("initializing gzip reader: %v", err)
+	}
+	defer gzr.Close()
+	if err := binary.ExpectBytes(gzr, []byte(csiMagic)); err != nil {
+		return nil, fmt.Errorf("checking magic: %v", err)
+	}
+
+	var minShift int32
```

Comment:
minShift is a detail of the implementation - it's best to talk about widths outside the implementation of the function

Reply:
done